packages/vue2-jest/lib/utils.js

```diff
@@ -68,23 +70,55 @@ const getBabelOptions = function loadBabelOptions(filename, options = {}) {
   return loadPartialConfig(opts).options
 }
 
+const tsConfigCache = {}
```

Comment:
Hm, how about using `Map` here? Perhaps a bit cleaner and more idiomatic?

```js
const tsConfigCache = new Map()

// later

```js
let tsConfig

if (tsConfig = tsConfigCache.get(path)) {
  return tsConfig
}
```

etc.